templates/create-daml-app/ui/src/components/LoginScreen.tsx.template

```diff
@@ -118,8 +119,10 @@ const LoginScreen: React.FC<Props> = ({onLogin}) => {
     (async function () {
       if (isLoading === false && isAuthenticated === true) {
         if (user !== undefined) {
+          const party = user["https://daml.com/ledger-api"];
           const creds: Credentials = {
-            party: user["https://daml.com/ledger-api"],
+            user: {userId: user.email ?? user.name ?? party, primaryParty: party},
```

Comment:
Where does the `.email` thing come from? I’m not quite clear on where we used that before.

Reply:
This will come from the `user` object Auth0 will return after successful authentication. I think whether it is set or not depends on how the user logged in, i.e. via setting up an account first or via OAuth2.

Reply:
makes sense thanks for the explanation